nestjs/src/courses/mentors/mentors.service.ts

```diff
@@ -134,9 +144,16 @@ export class MentorsService {
       resultScore: task.tr_score,
       solutionUrl: task.ts_url,
       taskDescriptionUrl: task.t_descriptionUrl,
+      status: task.tr_score ? StudentTaskSolutionItemStatus.Done : StudentTaskSolutionItemStatus.InReview,
+      endDate: this.getEndDate(task.ct_studentEndDate),
     }));
   }
 
+  private getEndDate(date: string) {
+    const endDate = Date.parse(date);
+    return new Date(endDate + twoWeeksMs);
```

Comment:
consider using `dayjs` which will remove this function and var `twoWeeksMs`

Reply:
Thank you for recommendation. I will look at `dayjs` in next PRs and refactor this function